pipeline.py

```diff
@@ -112,32 +91,32 @@ def pipeline(
             int
         ] = None,  # FIXME: https://issues.redhat.com/browse/RHOAIRFE-467
         sdg_base_model: str = "s3://<BUCKET>/<PATH_TO_MODEL>",
-        sdg_scale_factor: int = 2,  # Renamed upstream https://github.com/instructlab/instructlab/blob/f7d40f6ed5112d59132dd832bd332fa6fbbe7010/src/instructlab/configuration.py#L279-L290
-        sdg_pipeline: str = SDG_PIPELINE,
-        sdg_max_batch_len: int = MAX_BATCH_LEN,
-        sdg_sample_size: float = 1.0,
+        sdg_scale_factor: int = 30,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L125
```

Comment:
I think pointing to [d68c5a12026e08092bd4171a77667b09f7c36f72](https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72) is "risky". Why this hash in particular? I'd prefer to use a tag at best.

Reply:
That was just the most recent commit at `main` at the time me pulling the data (and still is as of now). Unless you lock a link to specific commit, it's not a permanent reference. Lines can shift in time and mapping to a tag/branch is a moving target. So linking to line e.g. 123 would point to a different property over time...

Reply:
Ok, but what's the issue with tag + line number? I think it's more natural and easier to understand since the tag will point to a given version. Thanks!

Reply:
Well.. tag is also moving target, but guess this is not abused that often. 😄 I think that is a good compromise. 🙂

Reply:
Sorry I don't mean to argue over this but I don't understand the "tag is also moving target", a tag is applied to a commit and considered immutable. While it’s technically possible to move or delete a tag, this is quite rare and generally discouraged.

Ok great for using a tag + line number which is a pretty stable and understandable reference :-)